coralproject/talk
# Feature/refine translation

## What does this PR do?
1. Refine the translations for Simplified and Traditional Chinese
2. Add i18n for 'Hide off-topic comments' in talk-plugin-offtopic plugin.

## How do I test this PR?
set TALK_DEFAULT_LANG to 'zh_TW' or 'zh_CN'
run the talk instance
see if everything in proper traditional Chinese or simplified Chinese
